### PR TITLE
slide show deliverable

### PR DESCRIPTION
**Pull request by SmithTM90**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5):4

What did you think of this deliverable?: I enjoyed this deliverable handling API calls, I found it to be easy enough to ease in to, but educational about api documentation, I will say that the Reddit API, while easy to access, can be kinda wonky. The AJAX part also felt educational, I had touched AJAX before, but this really helped drive some concepts home.

### CODE REVIEW

**Comment by scottheron** [comment]
Awesome job Travis. I'm impressed you went with another API also. Styling looks great too. Kudos getting the buttons to change for the appropriate context. I would remove the commented out code to clean things up and add comments to help others (and yourself) understand the code later. Well Done!

File: src/js/app.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+var picArray = [];
+var count = 0;
+
+$(document).ready(function(){
+  $('#inputForm').focus();
+
+  $('#searchForm').on('submit', function(e){
+    e.preventDefault();
+    $('#inputForm, #searchButton, #title, #description').addClass('hidden');
```
Comment (bhague1281) [inline src/js/app.js:9]: This is more-or-less what the `hide()` function does. Nice!

File: src/js/app.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      api_key: 'dc6zaTOxFJmzC'
+    }).done(function(data){
+      var array = data.data;
+
+      for(var i = 0; i < array.length; i++) {
+        var pic = array[i].images.original.url;
+        picArray.push(pic);
+      }
+    }).fail(function(data){
+
```
Comment (bhague1281) [inline src/js/app.js:23]: I think I left this out on Derek's code, but it's recommended to put something in these functions, whether it be a `console.log`, or more commonly, a message for the user

File: src/js/app.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    }).always(function(data){
+
+    });
+    $('#inputForm').val('');
+  });
+
+  $('#stopButton').on('click', function(e) {
+    e.preventDefault();
+    //location.reload() was kind of a cheeky way out, but it works and the code below it was giving me some issues, I may come back to it.
```
Comment (bhague1281) [inline src/js/app.js:32]: You're right. Thanks for letting me know 👍

**Pull request closed without merging**